src/main/java/star/exception/StarException.java

```diff
@@ -0,0 +1,48 @@
+package star.exception;
+
+/**
+ * Represents exceptions that are specific to the Star chatbot.
+ * Inherits from the Exception class.
+ */
+public class StarException extends Exception {
```

Comment:
Like the way you do the different exceptions. Its quite an interesting way to handle it